playwright.config.ts

```diff
@@ -61,36 +78,46 @@ const config: PlaywrightTestConfig<TestOptions> = {
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "5.21-edge",
+        storageState: "tests/.auth/user.json",
       },
+      dependencies: ["setup-chromium"],
```

Comment:
Wouldn't this run the `auth.setup.ts` suite a second time?

Reply:
In the local test scripts and in our workflows, we always run playwright tests with the `--project` flag for browser specific projects only. The setup projects only gets run once as a dependency to the browser specific projects.